trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/petrinet/unfolding/BranchingProcess.java

```diff
@@ -406,7 +404,7 @@ private boolean conflictPathCheck(final Condition<LETTER, PLACE> c1, final Condi
 	 * @return Set containing all Conditions and Events which are (strict) ancestors of a Condition. The dummyRoot is
 	 *         not considered as an ancestor.
 	 */
-	private Set<Object> ancestorNodes(final Condition<LETTER, PLACE> condition) {
+	public Set<Object> ancestorNodes(final Condition<LETTER, PLACE> condition) {

```

Comment:
What is the reason to make this public?

Reply:
I am pretty sure, I did not change this, at least not on purpose. I can revert it then.